### PR TITLE
Shape text

### PR DESCRIPTION
**Pull request by MatthewMeacham**
Created the Text shape based on what Hongchao Guan had done

### CODE REVIEW

**Review by shaminoo — APPROVED** [review]

**Review by allisoninman — APPROVED** [review]
As is, I approve this. [This PR](https://github.com/shaminoo/SmallBasic-Online/pull/7) will require that the following code be added to this class:

    public move(x: number, y: number): void {
        this.instance.x(x);
        this.instance.y(y);
    }

**Review by allisoninman — CHANGES_REQUESTED** [review]
See other comment, accidentally hit approve

**Comment by MicahElizabeth** [comment]
Shapes.SetText(name, text) Should be in this class
From Omar:
Basically you use it with Shapes.AddText().
If you added a text object, you should be able to change its contents.
For example:

myShapeObject = Shapes.AddText("hello world")
' now the text "hello world" is rendered on screen, and the variable myShapeObject holds its name (for example Text1).
Shapes.SetText(myShapeObject, " new world")
' now the shape would display the new string "new world"

File: src/app/shapes/Text.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as Konva from "konva";
+import { Shape } from "./Shape";
+
+export class Text extends Shape {
+
```
Comment (MicahElizabeth) [inline src/app/shapes/Text.ts]: I think Text is already a class, we might have to call this TextShape or something

Comment (MicahElizabeth) [inline src/app/shapes/Text.ts]: I think Text is already a class, we might have to call this TextShape or something

**Review by MicahElizabeth — CHANGES_REQUESTED** [review]

**Comment by MicahElizabeth** [comment]
I will work on these changes. 😃

**Review by MicahElizabeth — APPROVED** [review]

**Review by allisoninman — APPROVED** [review]
LGTM!

**Pull request merged**